onnxruntime/core/framework/provider_bridge_ort.cc

```diff
@@ -658,23 +658,6 @@ std::shared_ptr<IExecutionProviderFactory> CreateExecutionProviderFactory_Dnnl(i
   static ProviderLibrary library(LIBRARY_PREFIX "onnxruntime_providers_dnnl" LIBRARY_EXTENSION);
   if (!library.provider_)
     return nullptr;
-
-#if defined(_WIN32) && !defined(_OPENMP)
```

Comment:
>if defined(_WI [](start = 1, length = 14)

The PR lacks description and motivation. In particular, why this piece of code being removed?

Reply:
It's ok. This code was originally moved into the dnnl provider itself. My TensorRT change also modified this code and when I merged with master, github added it back in and I didn't notice. This change just fixes the incorrectly added back code. (they let me know about this already)

All good!

Reply:
This is part of #4738 which got reverted by Ryan by accident, as he mentioned. I had discussed it with him offline.